README.md

```diff
@@ -443,13 +443,13 @@ docker run --rm -it \
   --metadata_path=./model_artifacts/PATH_TO_METADATA_YAML
 ```
 
-## Syngen Installation Guide for MacOS ARM (M1/M2) with Python 3.10
+## Syngen Installation Guide for MacOS ARM (M1/M2) with Python 3.10 (3.11)
```

Comment:
Please, correct this title to "Syngen Installation Guide for MacOS ARM (M1/M2) with Python 3.10 or 3.11"